app/views/layouts/ucsdlib.html.erb

```diff
@@ -5,8 +5,9 @@
 		<%= render :partial => 'layouts/twitter_card' %>
 		<link href="<%= opensearch_catalog_path(:format => 'xml', :only_url => false) %>" title="<%= application_name%>" type="application/opensearchdescription+xml" rel="search"/>
 
-	  <script src="//libraries.ucsd.edu/jwplayer/jwplayer.js"></script>
-	  <script src="//libraries.ucsd.edu/jwplayer/key.js"></script>
+        <script src="//libraries.ucsd.edu/jwplayer/jwplayer.js"></script>
+        <script src="//libraries.ucsd.edu/jwplayer/key.js"></script>
+        <script src="https://www.mapquestapi.com/sdk/leaflet/v2.2/mq-map.js?key=zrnNkNZDkELMvViepTqFjZP8bEKxjDaG"></script>

```

Comment:
@mcritchlow I tried to use the env variable, however when I did the view source of the object, people still can see the key.  Attached is the screen shot.
![mapquestkey](https://cloud.githubusercontent.com/assets/1864660/16883548/5a39567e-4a79-11e6-9abf-1f831e972e87.png)